### PR TITLE
Add a use case driven guide for creating a credential store using elytron-tool for wildfly client configuration

### PR DESCRIPTION
**Pull request by PrarthonaPaul**
Supersedes https://github.com/wildfly-security/wildfly-elytron/pull/2068
preview: https://wildfly-security.github.io/wildfly-elytron/blog/credential-stores-on-client-side

### CODE REVIEW

File: _posts/2024-08-21-credential-stores-on-client-side.adoc [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+date: 2023-08-21
+tags: credential-store elytron-tool wildfly-config ejb
+synopsis: How to set up a credential store using the elytron-tool and use it for the client configuration on an application deployed to WildFly. 
+author: prarthonapaul
+---
+
+:toc: macro
+:toc-title:
+
+WildFly allows the use of credential stores to keep alias for sensitive information, such as, passwords for external services. Credential stores can be used to store different credentials under aliases and use credential-reference to specify them in server configuration. As a result, the credential is no longer visible in clear text. However, there are additional uses for credential store as well. This guides explains how to create credential stores using the elytron-tool and use it for an application deployed to WildFly.
```
Comment (fjuma) [inline _posts/2024-08-21-credential-stores-on-client-side.adoc]: s/alias/aliases

Comment (fjuma) [reply _posts/2024-08-21-credential-stores-on-client-side.adoc]: s/and use credential-reference/and we can then use credential-references

File: _posts/2024-08-21-credential-stores-on-client-side.adoc [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+WildFly allows the use of credential stores to keep alias for sensitive information, such as, passwords for external services. Credential stores can be used to store different credentials under aliases and use credential-reference to specify them in server configuration. As a result, the credential is no longer visible in clear text. However, there are additional uses for credential store as well. This guides explains how to create credential stores using the elytron-tool and use it for an application deployed to WildFly. 
+
+toc::[]
+
+== Prerequisite
+To follow along with this guide you will need:
+
+* about 10 minutes
+* WildFly with credential-store support
```
Comment (fjuma) [inline _posts/2024-08-21-credential-stores-on-client-side.adoc]: WildFly

File: _posts/2024-08-21-credential-stores-on-client-side.adoc [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+WildFly allows the use of credential stores to keep alias for sensitive information, such as, passwords for external services. Credential stores can be used to store different credentials under aliases and use credential-reference to specify them in server configuration. As a result, the credential is no longer visible in clear text. However, there are additional uses for credential store as well. This guides explains how to create credential stores using the elytron-tool and use it for an application deployed to WildFly. 
+
+toc::[]
+
+== Prerequisite
+To follow along with this guide you will need:
+
+* about 10 minutes
+* WildFly with credential-store support  
+* Elytron-tool for WIldFly server
```
Comment (fjuma) [inline _posts/2024-08-21-credential-stores-on-client-side.adoc]: Elytron Tool (provided with WildFly)

File: _posts/2024-08-21-credential-stores-on-client-side.adoc (hunk 1 above)
Comment (PrarthonaPaul) [reply _posts/2024-08-21-credential-stores-on-client-side.adoc]: fixed!

File: _posts/2024-08-21-credential-stores-on-client-side.adoc (hunk 2 above)
Comment (PrarthonaPaul) [reply _posts/2024-08-21-credential-stores-on-client-side.adoc]: fixed!

File: _posts/2024-08-21-credential-stores-on-client-side.adoc (hunk 3 above)
Comment (PrarthonaPaul) [reply _posts/2024-08-21-credential-stores-on-client-side.adoc]: fixed!

File: _posts/2024-08-21-credential-stores-on-client-side.adoc [hunk 4]
```diff
@@ -0,0 +1,158 @@
+---
+layout: post
+title: 'Using Credential Stores for WildFly Client'
+date: 2023-08-21
```
Comment (fjuma) [inline _posts/2024-08-21-credential-stores-on-client-side.adoc]: 2024 :)

**Comment by fjuma** [comment]
Thanks for the updates! Once the year is updated, this should be good to go.

**Pull request merged**